content/en/security/cloud_siem/log_detection_rules.md

```diff
@@ -176,9 +176,11 @@ Click the **Advanced** option to add queries that will **Only trigger a signal w
 
 ### Trigger
 
-{{< img src="security/security_monitoring/detection_rules/define_rule_case.png" alt="Define the rule case" >}}
+{{< img src="security/security_monitoring/detection_rules/define_rule_case2.png" alt="The set rule case section showing the default settings" style="width:80%;" >}}
 
-Rule cases, such as `a > 3`, are evaluated as case statements. Thus, the first case to match generates the signal. Click and drag your rule cases to manipulate their ordering.
+Enable **Create rules cases with the Then operator** if you want to generate a signal for the example, if query A occurs and then query B occurs.
```

Comment:
There's a small limitation that we may want to call out as well - "THEN operator can only be used in one case"

Also the feature is now GA so we should be good to merge 🙇🏽‍♀️